### ISSUE TITLE
MapControl in experimental release not working

### ISSUE THREAD

**Issue by nickrandolph** [microsoft/WindowsAppSDK#4163]
### Describe the bug

Is the MapControl in the [experimental release](https://learn.microsoft.com/en-us/windows/apps/windows-app-sdk/experimental-channel#version-15-experimental-150-experimental2) supposed to be working? All I'm seeing is a blue screen

![image](https://github.com/microsoft/WindowsAppSDK/assets/1614057/46d37ea6-7b45-40c8-84e3-89e199ea80dd)

### Steps to reproduce the bug

New WinUI application
Update to experimental release
Use following XAML in MainWindow

```
<Window
    x:Class="MapSample.MainWindow"
    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
    xmlns:local="using:MapSample"
    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
    mc:Ignorable="d">

    <Grid>
        <MapControl MapServiceToken="XXXXX" />
    </Grid>
</Window>
```

### NuGet package version

None

**Comment by dotMorten** [comment]
I hit the same problem. Tried both the token you'd use with UWP mapcontrol and the one you'd use with Azure maps (the latter which it looks like this control is just [wrapping the Azure Maps Javascript control](https://github.com/microsoft/microsoft-ui-xaml/blob/c4e32a2727e2cdcbf3bdf14f8dfe38282dcecd4f/controls/dev/MapControl/map.html#L5-L7) in a [webview](https://github.com/microsoft/microsoft-ui-xaml/blob/c4e32a2727e2cdcbf3bdf14f8dfe38282dcecd4f/controls/dev/MapControl/MapControl.xaml#L11-L15)).

**Comment by dotMorten** [comment]
Looks like `initializeMap` isn't called. I got things working with this workaround:
```cs
 public MainWindow()
 {
     this.InitializeComponent();
     mapControl.MapServiceToken = mapServiceToken;
     mapControl.Loaded += MapControl_Loaded;
 }

 private void MapControl_Loaded(object sender, RoutedEventArgs e)
 {
     var webView = VisualTreeHelper.GetChild(mapControl, 0) as WebView2;
     webView.NavigationCompleted += WebView_NavigationCompleted;
 }

 private async void WebView_NavigationCompleted(WebView2 sender, Microsoft.Web.WebView2.Core.CoreWebView2NavigationCompletedEventArgs args)
 {
     sender.NavigationCompleted -= WebView_NavigationCompleted;
     var result = await sender.ExecuteScriptAsync($"initializeMap(0,0,\"{mapControl.MapServiceToken}\");");
 }
```

**Comment by hez2010** [comment]
BTW I would like to expect we can have a native MapControl just like what the UWP MapControl was, instead of the current WebView based garbage.

**Comment by devsko** [comment]
It kind of works for me after upgrading to version `1.5.240205001-preview1`. Unfortunately, this kind of map is far from being an adequate replacement for the UWP MapControl.

**Comment by eduardobragaxz** [comment]
> BTW I would like to expect we can have a native MapControl just like what the UWP MapControl was, instead of the current WebView based garbage.

I was taken aback by that. It makes no sense.

**Comment by dotMorten** [comment]
@eduardobragaxz It's probably worth opening an issue specifically discussing this rather than bringing it up on this unrelated issue

**Comment by eduardobragaxz** [comment]
@dotMorten you're right, my bad. I opened a discussion in the WinUI repo: https://github.com/microsoft/microsoft-ui-xaml/discussions/9323
